Wait for screen change before showing log in yast2_ntplient

kravciak opened the pull request:
alt-l is send before yast loads window after previous alt-o

Requires: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/221
Fixing: https://openqa.suse.de/tests/588769#step/yast2_ntpclient/21
Local run: http://dhcp91.suse.cz/tests/2574

okurz commented:
A plain `wait_still_screen` will wait for 7 seconds regardless if the screen is already static. What about making this an assert_screen or `wait_still_screen(1)`?

kravciak commented:
Guess I got lazy as yast tests are not very "effective".
1s might not be enough under load. I added needle (see description) and cleaned up some code.
